### ISSUE TITLE
New Example: Kendo UI

### ISSUE THREAD

**Issue by tirumalavasu** [formly-js/angular-formly-website#58]
I would like an example that demonstrates:

Kendo UI Form with validations just shown as in below demo

http://demos.telerik.com/kendo-ui/validator/angular

**Comment by burkeholland** [comment]
Just to clarify, you're looking for an example of using Formly with the Kendo UI Validator?

**Comment by tirumalavasu** [comment]
yes i need for both validation and how to use kendo ui angular directives with angular formly?

**Comment by thinw002** [comment]
There's a good example here https://stackoverflow.com/questions/38293270/how-to-set-custom-template-for-kendo-ui-multiselect-with-angular-formly
